src/common/low_precision_transformations/include/low_precision/layer_transformation.hpp

```diff
@@ -300,7 +300,7 @@ class LP_TRANSFORMATIONS_API LayerTransformation : public ov::pass::MatcherPass
     };
 
     LayerTransformation(const Params& params);
-    virtual ~LayerTransformation() = default;
+    virtual ~LayerTransformation();
```

Comment:
It looks like dtor definition is missing now. That's most likely the cause of linker error in ubuntu builds.